docker-compose.prod.yml

```diff
@@ -4,7 +4,7 @@ services:
   app:
     container_name: app
     image: ${DOCKER_REGISTRY}/${DOCKER_REPOSITORY}:${DOCKER_TAG}
-    command: "sh /app/wait-for-db.sh $DGRAPH_HEALTH_CHECK_ENDPOINT node /app/dist/src/main.js"
```

Comment:
Yup, we don't need to wait cause we're going to use Amazon RDS, so postgres won't be a part of docker-compose cluster and it will be running all the time :)